lib/dynamoid/adapter.rb

```diff
@@ -19,7 +19,7 @@ def tables
       if !@tables_.value
         @tables_.swap{|value, args| benchmark('Cache Tables') {list_tables}}
       end
-      @tables_.value
+      @tables_.value || []
```

Comment:
sorry about being so slow to get back. I'll take a look tonight by bootstrapping a new project with the same dependencies and report back with the errors I saw originally. The error posted by @cavi21 doesn't look like the error I originally saw. The exception thrown was when .include? was being called on the tables method. I'd get .include? can't be called on nil.